peoplesoft/tests/functional/cs/pages/EO_ADDR_USA_SEC.ts

```diff
@@ -0,0 +1,345 @@
+import intern from "intern";
+import { spawn } from "child_process";
+import pollUntil from "@theintern/leadfoot/helpers/pollUntil";
+
+const { registerSuite } = intern.getInterface('object');
+const { assert } = intern.getPlugin('chai');
+
+const root = __dirname + "../../../../..";
+const PRO_WEB_AUTH_TOKEN = process.env.PRO_WEB_AUTH_TOKEN;
+const GLOBAL_INTUITIVE_AUTH_TOKEN = process.env.GLOBAL_INTUITIVE_AUTH_TOKEN;
+
+function addGlobalIntuitive() {
```

Comment:
At some point in the future this entire chain could probably be abstracted into its own function.